app/models/project.rb

```diff
@@ -65,19 +65,19 @@ def send_mail
   end
 
   def self.tagged_with(name)
-    Tag.find_by!(name: name).projects
+    Tag.find_by!(name: name).projects.public_view
```

Comment:
Keep public scope where we need ie in the controller action, What if need to fetch private projects of a user with a particular tag in future ?.

Reply:
yeah makes sense I'll change this asap.